src/OpenLoco/Localisation/StringManager.h

```diff
@@ -135,6 +136,11 @@ namespace OpenLoco
 
 namespace OpenLoco::StringManager
 {
+    const uint8_t kUserStringSize = 32;
+
+    const uint16_t kUserStringsStart = 0x8000;
+    const uint16_t kUserStringsEnd = kUserStringsStart + Limits::kMaxUserStrings;
+
```

Comment:
It's fine to expose `kUserStringSize`, but please revert the change to `kUserStringsStart` and `kUserStringsEnd`.